### ISSUE TITLE
DropdownItem onSelect callback not invoked when clicking off the text

### ISSUE THREAD

**Issue by BoykoAlex** [pivotal-cf/pivotal-ui#505]
First of all `onClick` didn't work for the dropdown item. Hence, I was forced to use `onSelect` which i found out about by looking at issue: https://github.com/pivotal-cf/pivotal-ui/issues/496

I have the following dropdown:
```
<Dropdown flat floatMenu title={user.name}>
  <DropdownItem href="#" onSelect={this.logout.bind(this)} >Sign out</DropdownItem>
</Dropdown>
```
The `onSelect` callback is only invoked when i click on the text. If I click outside the text `<a>` element but on the enclosing `li` then nothing happens.

**Comment by charleshansen** [comment]
we do have `onClick` now for dropdown items in 8.3.0. There may be a css fix for `onSelect`, but I'm worried it may have bad side effects. We'll look into it.

**Comment by elenasharma** [comment]
This fix for this issue has been committed and should be published in the next release.

**Comment by jberney** [comment]
The [fix](https://github.com/pivotal-cf/pivotal-ui/commit/e62e3e7e9cfce9f1c13fbe8fe7912aeda228fb81) has been published in the latest release: **v8.3.2**.
